libraries/bot-connector/src/test/java/com/microsoft/bot/connector/authentication/ParameterizedBotFrameworkAuthenticationTest.java

```diff
@@ -0,0 +1,205 @@
+package com.microsoft.bot.connector.authentication;
+
+import com.microsoft.bot.connector.Channels;
+import com.microsoft.bot.schema.Activity;
+
+import com.microsoft.bot.schema.ChannelAccount;
+import com.microsoft.bot.schema.RoleTypes;
+import okhttp3.OkHttpClient;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+import static org.junit.Assert.*;
+
+public class ParameterizedBotFrameworkAuthenticationTest {
+    @Test
+    public void getOriginatingAudience_shouldBeCorrectlyFilled() {
+        PasswordServiceClientCredentialFactory passwordServiceClientCredentialFactory = new PasswordServiceClientCredentialFactory("42",
+            "test");
+
+        AuthenticationConfiguration authenticationConfiguration = new AuthenticationConfiguration();
+        assertEquals("test@example.org",
+            (new ParameterizedBotFrameworkAuthentication(true, "test@example.org", "test@example.org", "ABC123",
+                "https://example.org/example", "test@example.org", "test@example.org", "42", passwordServiceClientCredentialFactory,
+                authenticationConfiguration, new OkHttpClient())).getOriginatingAudience());
```

Comment:
Can we move `test@example.org` to a variable / constant and reference it twice? 🙂

![image](https://user-images.githubusercontent.com/58704965/130090756-bb810c90-ea08-433b-9c12-5ac893f66430.png)